lib/blockchain_double.js

```diff
@@ -123,7 +123,7 @@ BlockchainDouble.prototype.initialize = function(accounts, callback) {
   });
 };
 
-BlockchainDouble.prototype.createVMFromStateTrie = function(state) {
+BlockchainDouble.prototype.createVMFromStateTrie = function(state, activatePrecompiles) {
```

Comment:
Thoughts on using a default argument?  It would simplify line 149 too.
`function(state,activatePrecompiles = false)`